packages/@webex/plugin-meetings/test/unit/spec/locus-info/lib/SeqCmp.json

```diff
@@ -517,6 +517,22 @@
       "new": "seq4g",
       "result": "DESYNC",
       "description": "New seq is greater than current, but both current and new base have some unique entries."
+    },
+    "updt25": {
+      "current": "seq4",
+      "newbase": "seq5c",
```

Comment:
minor thing, but shouldn't `new` be greater than `newbase`?

Reply:
It *is* later.  Don't look at me, I didn't name the sequences. :-)